src/containers/ConfirmationContainer/ConfirmationContainer.vue

```diff
@@ -156,9 +159,9 @@ export default {
     });
   },
   created() {
-    this.$eventHub.$on('showSuccessModal', (message, linkMessage) => {
+    this.$eventHub.$on('showSuccessModal', (message, linkMessage, linkTo) => {
       if (!message) message = null;
-      this.showSuccessModal(message, linkMessage);
+      this.showSuccessModal(message, linkMessage, linkTo);
```

Comment:
Please make sure to do this on all instances of `showSuccessModal`

Reply:
I shouldn't change linkTo value. I added a new prop "etherscanLink" to pass etherscan URL.

Reply:
Yeah. I tested on Real network and Ropsten Testnet and they all look good!!!